src/admin-toolkit-ui/index.tsx

```diff
@@ -284,7 +282,13 @@ function initSmartItemsSync(engine: IEngine, sdkHelpers?: ISDKHelpers) {
   ]
 
   const smartItems = getSmartItems(engine)
-  syncEntitiesComponents(engine, smartItems, requiredComponentIds, sdkHelpers)
+  syncEntitiesComponents(
+    engine,
+    smartItems,
+    requiredComponentIds,
+    getNextEnumEntityId(engine),
```

Comment:
i'm not sure about this. This getNextEnumEntityId will be different for each user, meaning that they will reference to different entities 🤔